test/NuGet.Clients.Tests/NuGet.PackageManagement.UI.Test/Actions/AccessiblePackageIdentityTests.cs

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using NuGet.Packaging.Core;
+using NuGet.Versioning;
+using Xunit;
+
+namespace NuGet.PackageManagement.UI.Test
+{
+    public class AccessiblePackageIdentityTests
+    {
+        [Fact]
+        public void AutomationName_WhenCultureIsNeutral_IdVersionConstructor_ReturnsMessage()
+        {
+            var result = new AccessiblePackageIdentity("test.package", new NuGetVersion(0, 0, 1));
+
+            Assert.Equal("test.package version 0.0.1", result.AutomationName);
```

Comment:
Probably fine... but maybe `Version.ToNormalizedString()` would be better in case we change how that looks at some point?

Reply:
Good thought. If the implementation of `Version.ToNormalizedString()` changes, then, the test should fail to indicate that the accessible name needs to be addressed. I'm thinking of making 2 assertions, one for the hardcoded string (actual) and one for comparing against `Version.ToNormalizedString()`